lib/v0_bindings_extensions.rb

```diff
@@ -165,6 +130,38 @@ def summarize(user_uuid:)
     Api::V0::Bindings::GetHighlightsSummaryParameters
   ].each do |klass|
     klass.class_exec do
+
+      def get_highlights(user_id)
+        filters = to_hash.except(:page, :per_page, :sets)
+        filters[:user] = []
+
+        if (sets.blank? or sets.include?('user:me')) and user_id.present?
```

Comment:
normally better to use `||` and `&&` instead of `or` and `and` because the former have higher operator precedence than the latter.  Here it won't have an impact b/c there are no other operators between the boolean operators, but if there were the lower precedence of `or` and `and` could cause unexpected behavior.